user-docs/latest/content-manager/introduction-to-content-manager.md

```diff
@@ -0,0 +1,100 @@
+# Introduction to the Content Manager
+
+The Content Manager is a core plugin of Strapi. It is a feature that is always activated by default and cannot be deleted. It is accessible both when the application is in a development and production environment.
+
+The Content Manager is divided into 2 categories, both displayed in the main navigation: *Collection types* and *Single types*. Each category contains the available collection and single content-types, which were created beforehand using the Content-Types Builder. From these 2 categories, admin panel users can create, manage, and publish content.
+
+::: tip 💡 TIP
+Click the search icon <Fa-Search /> in the main navigation to use a text search and find one of you content types more quickly!
+:::
+
+## Collection types
+
+The *Collection types* category of the Content Manager displays the list of available collection types which are directly accessible from the main navigation of the admin panel.
+
+For each available collection type multiple entries can be created which is why each collection type is divided into 2 interfaces: the list view and the edit view (see [Writing content](writing-content.md)).
+
+The list view of a collection type displays all entries created for that collection type.
+
+![List view of a collection type in the Content Manager](../assets/content-manager/content-manager_list-view.png)
+
+From the list view, it is possible to:
+
+- make a textual search (1) or set filters (2) to find specific entries,
+- create a new entry (3),
+- configure the fields displayed in the table of the list view (4).
+
+::: tip 💡 TIP
```

Comment:
I saw you added some information about the "Sorting"
Here and at the bottom of the page.
I think having a full section to talk about it could be a better solution I think.

Reply:
I removed the full note about relational fields from the introduction and instead put a link to "Configuring view" where you can find all information regarding relational fields.
That way, we also don't have the problem of talking about sorting at 2 different spots in the documentation!